src/test/java/com/artipie/asto/S3StorageTest.java

```diff
@@ -91,6 +91,25 @@ void shouldNotExistForUnknownObject(final AmazonS3 client) {
         MatcherAssert.assertThat(exists, Matchers.equalTo(false));
     }
 
+    @Test
+    void shouldMoveObject(final AmazonS3 client) throws Exception {
```

Comment:
@olegmoz We should add a test assuring that the object is ~moved~, not ~copied~. Please add a new test assuring that `source` object is removed from bucket.

Reply:
@paulodamaso done